guide/06-imagery-and-raster-analysis/creating_imagery_layers.ipynb

```diff
@@ -0,0 +1,474 @@
+{
```

Comment:
<p>In this guide, you've learned how to create imagery layers from your local raster data or data in your datastore using the copy_raster and create_image_collection functions. These functions provide a simple way of creating an imagery layer from a normal raster dataset, multidimensional datasets, and even satellite datasets. You have also seen how to create an imagery layer from either a single or multiple raster datasets. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1036/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>